__init__.py

```diff
@@ -251,14 +254,44 @@ def match(self):
 
     def remove(self):
         '''remove()
-
-        From HDF5 file remove group paired with this variable.
-
+        Removes associated HDF5 group.
```

Comment:
I would keep the first sentence "From HDF5 file remove group paired with this variable."